document/workflow/variable-tree/data-source/skeleton-tree-data-source.ts

```diff
@@ -116,6 +116,14 @@ export class SkeletonTreeDataSource implements TreeNodeFactory, TreeNodeObserver
         return this.describer;
     }
 
+    set uncollapseWhileMatching(value: boolean) {
```

Comment:
Then it will only uncollapse for the first formula. What if the formula area contain more then one formula, needing the datasource to match and uncollapse?
For Example you have a mapping with
`%0%=%1%`
`%0%.text=%1%.text`
In this case should not only the first match `%0%` should do an uncollapsing.

Reply:
Too bad. Unfortunately, there's no way for the data source to find out if it's the initial or any other matching round. You have to tell it from the outside.

Reply:
I would like that more respecting the data source's API. On the other hand it makes calling `processVariable`/`addMatchingStructure` more complicated and thus harder to understand.
It's down to you to decide.